src/transformers/modeling_flax_utils.py

```diff
@@ -0,0 +1,146 @@
+import os
+from abc import ABC, abstractmethod
+from pickle import UnpicklingError
+from typing import Dict
+
+from flax.linen import Module
+from flax.serialization import to_bytes
+from flax.traverse_util import unflatten_dict
+from jax.random import PRNGKey
+from transformers import PretrainedConfig, logger
+from transformers.file_utils import WEIGHTS_NAME, cached_path, hf_bucket_url, is_remote_url
+
+
+class FlaxPreTrainedModel(ABC):
```

Comment:
Even if the property `config` is overridden in every child, it seems like a necessary attribute for every `FlaxPreTrainedModel` to me, so it should be in here as well, shouldn't it?